### PR TITLE
PATCH: changing to font icons as a temporary fix to style weather icons

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
We were using SVG's as background images on our weather components, however this meant that the colors of the icons couldn't be changed. 
- We have moved to use Meteocons hosted on hungry geek (for the moment)
- We will eventually use our own icons SVGs hosted on a CDN or in the HTML (for over-brandng)

#### What tests does this PR have?
Nothing new

#### How can this be tested?
- run `grunt build`
- run grunt `docs`
- have a play with the weather component (you can edit int eh code box of the docs, it doesn't look like it but you can)
- Change the font color of the icon in CSS

#### Screenshots / Screencast
screen shot 2015-09-11 at 13 15 36

#### What gif best describes how you feel about this work?
Face Palm

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.

#### Software Engineer or Developer review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked all the tests are passing.

#### Software Engineer or project guru review:
- [ ] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [ ] I’ve checked for coding anti-patterns.
- [ ] I’ve checked for appropriate test coverage.
- [ ] I’ve checked all the tests are passing.

### CODE REVIEW

**Comment by hpoom** [comment]
"We will eventually use our own icons fonts hosted on a CDN" - Would we not eventually move to SVG? I would assume we want to sunset all use of icon fonts in favour of SVG?

**Comment by jodiedoubleday** [comment]
Yup wrong wording, changed in PR decription

**Comment by cameronviner** [comment]
Taking dev

**Comment by cameronviner** [comment]
Looks pretty solid to me :+1: https://cloudup.com/c6wAqyr8XRU

**Comment by jackdcrawford** [comment]
Review :raised_hands:

File: src/components/weather/code/less/_weather.less [hunk 1]
```diff
@@ -2,24 +2,35 @@
     #Weather
 \*------------------------------------*/
 
+@font-face {
+    font-family: 'MeteoconsRegular';
```
Comment (jackdcrawford) [inline src/components/weather/code/less/_weather.less]: **linting comment:** 4 spaces

File: docs/public/assets/fonts/meteocons-webfont.svg [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+<glyph unicode="&#x2006;" horiz-adv-x="320" />
+<glyph unicode="&#x2007;" horiz-adv-x="320" />
+<glyph unicode="&#x2008;" horiz-adv-x="240" />
+<glyph unicode="&#x2009;" horiz-adv-x="384" />
+<glyph unicode="&#x200a;" horiz-adv-x="106" />
+<glyph unicode="&#x202f;" horiz-adv-x="384" />
+<glyph unicode="&#x205f;" horiz-adv-x="480" />
+<glyph unicode="&#xe000;" horiz-adv-x="500" d="M0 0z" />
+</font>
+</defs></svg>
```
Comment (jackdcrawford) [inline docs/public/assets/fonts/meteocons-webfont.svg]: **linting comment:** no EOF line

File: src/components/weather/code/less/_weather.less [hunk 3]
```diff
@@ -27,13 +38,18 @@
...
-  overflow: hidden;
+  padding-bottom:@padding-vertical-small;
+  &:before {
+    font-family: 'MeteoconsRegular';
+    content:"@{weather}";
+    font-size:3.5em;
+    visibility: visible;
+  }
+  span {
+    display:block;
```
Comment (jackdcrawford) [inline src/components/weather/code/less/_weather.less]: **linting comment:** inconsistent spaces between property & value here

**Comment by jackdcrawford** [comment]
Couple of really minor linting things, code looks good though :+1:.

Looking forward to getting it into HEHA!

rainbow

**Comment by jackdcrawford** [comment]
:+1:

**Pull request merged**